support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -52,29 +52,14 @@ const pageQaId = 'qa-paper-subscriptions';
 function PaperLandingPage({
 	productPrices,
 	promotionCopy,
-	participations,
 }: PaperLandingPropTypes) {
 	const sanitisedPromoCopy = getPromotionCopy(promotionCopy);
 
-	const isNationalDeliveryAbTestVariant =
-		participations.nationalDelivery === 'variant';
-
-	const [shouldRedirectToHomeDelivery, setShouldRedirectToHomeDelivery] =
```

Comment:
if there are links from elsewhere on the web to this page and we want to show home delivery first maybe we should keep doing this? not sure

Reply:
I _think_ this is fine?

Reply:
have discussed with @GHaberis and since the top Google results are to `paper` not `paper/delivery` we're going to keep the on page redirect